packages/backend/server/src/plugins/copilot/providers/anthropic.ts

```
@@ -211,6 +211,12 @@ export class AnthropicProvider
             }
             break;
           }
+          case 'tool-call': {
```

Biome (1.9.4)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)